NEWS.md

```diff
@@ -2,6 +2,14 @@
 
 ### Changed
 
+* rename `select_causal_rules()` function in `lasso_rules_filter()` and 
+update its definition
+* add parameter `replace` to set replacement in bootstrapping for rules generation
```

Comment:
In this one: rules generation now accepts `replace` parameter to set replacement in bootstrapping.